docs/src/modules/utils/getDemoConfig.js

```diff
@@ -81,17 +81,23 @@ function getLanguageConfig(demoData) {
 
 export default function getDemo(demoData) {
   const baseConfig = {
-    title: 'Material demo',
+    title: demoData.title,
     description: demoData.githubLocation,
     files: {
-      'index.html': `
-<body>
-  <!-- Fonts to support Material Design -->
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
-  <!-- Icons to support Material Design -->
-  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
-  <div id="root"></div>
-</body>
+      'public/index.html': `
```

Comment:
See https://github.com/codesandbox/codesandbox-client/issues/4737. Old sandboxes are fine since they only use `body`. Once we want to use `head` we need to have it in `public/index.html`. 

Benefit: Now also works when downloaded though we've never heard anyone complain.